kratos/processes/variational_distance_calculation_process.h

```diff
@@ -476,43 +470,42 @@ class VariationalDistanceCalculationProcess : public Process
             CalculateNormDxFlag);
 
         // TODO: check flag DO_EXPENSIVE_CHECKS
-        mp_solving_strategy->Check();
+        mpSolvingStrategy->Check();
     }
 
-    virtual void ReGenerateDistanceModelPart(ModelPart& r_base_model_part)
+    virtual void ReGenerateDistanceModelPart(ModelPart& rBaseModelPart)
     {
         KRATOS_TRY
 
-        Model& current_model = r_base_model_part.GetModel();
-        if(current_model.HasModelPart( mAuxModelPartName ))
-            current_model.DeleteModelPart( mAuxModelPartName );
+        if(mrModel.HasModelPart( mAuxModelPartName ))
+            mrModel.DeleteModelPart( mAuxModelPartName );
```

Comment:
why not call `Clear` here?

Reply:
`ModelPart.Clear()`?

Reply:
I meant `VariationalDistanceCalculationProcess.Clear()`
not sure if it possible since that one also clears the Strategy